src/Files.Core.Storage/Contracts/ITrashWatcher.cs

```diff
@@ -15,11 +15,6 @@ public interface ITrashWatcher : IWatcher
 		/// </summary>
 		event EventHandler<SystemIO.FileSystemEventArgs>? ItemDeleted;
 
-		/// <summary>
-		/// Gets invoked when an item changing is detected by the watcher
-		/// </summary>
-		event EventHandler<SystemIO.FileSystemEventArgs>? ItemChanged;
```

Comment:
Why was this removed?

Reply:
I'm not sure why added.
In fact, it isn't needed.
But maybe needed in other platform. Should we keep?

Reply:
Let's keep it

Reply:
Done